at_onboarding_flutter/example/lib/main.dart

```diff
@@ -15,8 +13,6 @@ void main() {
   runApp(const MyApp());
 }
 
-final StreamController<ThemeMode> updateThemeMode = StreamController<ThemeMode>.broadcast();
```

Comment:
I have a feature which allow switching theme dark/light here.
This PR will remove feature of example.
Could you please check again?

Reply:
@sonle-geekyants  But if you check your PR of switching theme dark/light it only contain the main.dart file other files are missing that is the reason why we are reverting it.

Reply:
@Intiserahmed It's changed themeMode of MaterialApp and all child/inherited widget will get theme from parent.
I just want user known that OnBoarding also support for dark theme.

Reply:
@sonle-geekyants I understand but the Changes you made didn't take in the PR you did. So can you look into your PR.

Reply:
for time being can we merge this or wait?

Reply:
Merge

Reply:
Can we wait for a review from Sarika? the commit 64ab3a3 belongs to her. It's better if she knows about these changes.
Thank you all!